FoxHound.Client/src/Blogs/Blogs.tsx

```diff
@@ -58,11 +58,11 @@ const Blogs: React.FC = () => {
         </ul>
       )}
 
-      <AddBlog handleBlogAdded={handleBlogAdded} />
```

Comment:
I think you can remove the AddBlog from the Blogs listing. It might make sense, now that we have navigation, to just be a link in the nav bar.